drivers/snowflake/snowflake.go

```diff
@@ -25,7 +25,7 @@ func (sf *Snowflake) Analyze(s *schema.Schema) error {
 	}
 	s.Driver = d
 
-	tableRows, err := sf.db.Query(`SELECT table_name, table_type, comment FROM information_schema.tables WHERE table_schema = ?`, s.Name)
+	tableRows, err := sf.db.Query(`SELECT table_name, table_type, comment FROM information_schema.tables WHERE table_schema = ? order by table_name`, s.Name)
```

Comment:
Is it possible to have the tables in order of creation instead of in order of table name?

Is there a column in information_schema.tables to achieve the above in order of creation?

Reply:
@k1LoW 
MySQL has a creation date column, but it doesn't seem to exist in PostgreSQL

https://dev.mysql.com/doc/refman/8.0/ja/information-schema-tables-table.html
https://www.postgresql.jp/document/13/html/infoschema-tables.html

Reply:
@wate 
Thank you.
However, the target of this pull request seems to be Snowflake.

Reply:
@k1LoW 

I believe Snowflake allows sorting by creation date on the `created` column
https://docs.snowflake.com/ja/sql-reference/info-schema/tables.html

Reply:
@wate Wow!! Thank you!!!

Reply:
> Is it possible to have the tables in order of creation instead of in order of table name?

Why would you want to do that?  I desire a consistent listing of tables in alphabetical order.

Reply:
If it is in the order of creation, there is a choice to make them in the order intended by the database creator.

> I desire a consistent listing of tables in alphabetical order.

Maybe setting [`format.sort:`](https://github.com/k1LoW/tbls#document-format
) will give you the display you are looking for.

Reply:
@awhitford-cip @wate 
If sorting by created is selected, the actual order of table creation is unique. However, I found that depending on the speed of creation, created may be at the same time, and the display order may not be unique.

So, I reconsidered that the only way to sort is by table name

Reply:
Right now, my experience is that the table ordering is basically random because it is not defined.  So running tbls against the same database with no material database changes can result in different results, which makes it very difficult to find the table you are interested in when the list is random.

Looking forward to this fix.